core/src/main/java/org/mobilitydata/gtfsvalidator/notice/Notice.java

```diff
@@ -33,6 +34,14 @@ public Notice(Map<String, Object> context) {
     return Collections.unmodifiableMap(context);
   }
 
+  public SeverityLevel getSeverityLevel() {
+    return this.severityLevel;
+  }
+
+  public void setSeverityLevel(SeverityLevel severityLevel) {
+    this.severityLevel = severityLevel;
+  }
```

Comment:
I am unsure if we should make Notices mutable. Let's just have two constructors:

* public Notice(Map<String, Object> context, SeverityLevel severityLevel)
* public Notice(Map<String, Object> context) - defaults to ERROR. This constructor may be removed later during refactoring.